dev/com.ibm.websphere.appserver.jsp-2.2/com.ibm.websphere.appserver.jsp-2.2.feature

```diff
@@ -49,5 +49,5 @@ Subsystem-Name: JavaServer Pages 2.2
 -files=dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.jsp_1.0-javadoc.zip
 kind=ga
 edition=core
-IBM-Maven-Dependency: javax.servlet.jsp:javax.servlet.jsp-api:2.3.1, \
- javax.el:el-api:3.0.0
\ No newline at end of file
+IBM-Maven-Dependency: javax.servlet.jsp:javax.servlet.jsp-api:2.2.1, \
+ javax.el:el-api:2.2
```

Comment:
You can remove javax.el:el-api:2.2 or chang eit to javax.el:el-api:2.2.1 (see https://mvnrepository.com/artifact/javax.servlet.jsp/javax.servlet.jsp-api/2.2.1)